testdata/ansible/memcached-operator/PROJECT

```diff
@@ -1,7 +1,12 @@
 domain: example.com
 layout: ansible.sdk.operatorframework.io/v1
 plugins:
-  manifests.sdk.operatorframework.io/v2: {}
+  manifests.sdk.operatorframework.io/v2:
+    resources:
+    - domain: example.com
+      group: cache
+      kind: Memcached
+      version: v1alpha1
```

Comment:
@Adirio For we move forward here I do not think that we should track the info. Can we keep it working as before?